src/main/java/com/solr/clientwrapper/domain/service/SolrSchemaService.java

```diff
@@ -46,10 +46,10 @@
 	private static String DEFAULT_SOLR_CLOUD_COLLECTION = "gettingstarted1";
 	@Value("solr.client.url.static")
 	private static String TEST2;
-	private static String URL_STRING = "http://localhost:8985/solr/";
+	private static String URL_STRING = "http://localhost:8984/solr/";
```

Comment:
Take the URLs from configuration file. Then, only in config file, we can change the port